<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1518</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="login_sign_pages">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>491</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0.0817308 rgba(230, 136, 22, 255), stop:0.240385 rgba(206, 107, 8, 255), stop:0.423077 rgba(230, 136, 22, 255), stop:0.586538 rgba(206, 107, 8, 255), stop:0.802885 rgba(230, 136, 22, 255), stop:0.947115 rgba(206, 107, 8, 255));

border-color: rgb(194, 102, 36);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="login_page">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_l_user">
      <property name="geometry">
       <rect>
        <x>186</x>
        <y>100</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
border-color: rgb(13, 13, 13);</string>
      </property>
     </widget>
     <widget class="QLabel" name="user_l_l">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jokerman</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string> Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="pass_l_l">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jokerman</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string> Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_l_pass">
      <property name="geometry">
       <rect>
        <x>186</x>
        <y>210</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enter_butt">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>320</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jokerman</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>true</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sign_page">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="boss_l">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jokerman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string> boss name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_boss">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="username_l">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jokerman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string> username</string>
      </property>
     </widget>
     <widget class="QLabel" name="pass_l">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jokerman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string> password</string>
      </property>
     </widget>
     <widget class="QLabel" name="confirm_l">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jokerman</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string> confirm pass</string>
      </property>
     </widget>
     <widget class="QLabel" name="email_l">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jokerman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string>   E_Mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_user">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_pass">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_confirm">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_email">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="create_butt">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>350</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Jokerman</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>true</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>630</y>
      <width>841</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk {background-color: #488704;
 width: 10px;
 margin: 0.5px;
}
 QProgressBar{text-align: center;}

</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>1031</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_butt">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>390</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1189</x>
      <y>10</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>progressBar</zorder>
   <zorder>play_butt</zorder>
   <zorder>login_sign_pages</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1518</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
